nl.js

```diff
@@ -0,0 +1,747 @@
+settings.interface.hideNames.team Alleen Team
+settings.interface.hideNames.enemy Alleen Tegenstanders
+settings.interface.hideNames.all Iedereen
+settings.interface.nametagStyle Naam Stijl
+settings.interface.nametagOpac Naam Ondoorzichtigheid
+settings.interface.nametagStyle.health Alleen Gezondheid
+settings.interface.nametagStyle.name Alleen naam
+settings.interface.nametagStyle.nameLvl Alleen naam en level
+settings.interface.nametagStyle.all Alles
+settings.interface.healthNum Naam Gezondheid Nummer
+settings.interface.healthColT Naam Gezondheid Kleur Team
+settings.interface.healthColE Naam Gezondheid Kleur Tegenstanders
+settings.interface.xpBarCol XP Bar Kleur
+settings.interface.xpBarOpac XP Bar Doorzichtigheid
+settings.crosshair.header Crosshair
+settings.crosshair.type Soort
+settings.crosshair.type.2 Custom
+settings.crosshair.type.3 Gelaagd
+settings.crosshair.type.4 Afbeelding
+settings.crosshair.type.5 Precisie
+settings.crosshair.style Stijl
+settings.crosshair.style.0 Kruis
+settings.crosshair.style.1 Holle Cirkel
+settings.crosshair.style.2 Vaste Cirkel
+settings.crosshair.style.3 Holle Vierkant
+settings.crosshair.style.4 Vaste Vierkant
+settings.crosshair.image Afbeelding
+settings.crosshair.image.paste Crosshair Afbeelding
+settings.crosshair.always Laat altijd zien
+settings.crosshair.color Kleur
+settings.crosshair.shadow.color Schaduw Kleur
+settings.crosshair.shadow.thickness Schaduw dikheid
+settings.crosshair.size Grootte
+settings.crosshair.thickness Dikte
+settings.crosshair.gap Gat
+settings.crosshair.dot Punt
+settings.crosshair.opacity.use Gebruik Eigen Ondoorzichtigheid
+settings.crosshair.opacity.custom Eigen Ondoorzichtigheid
+settings.hitmarker.header Hitmarker
+settings.hitmarker.color Kleur
+settings.hitmarker.kcolor Killkleur
+settings.hitmarker.thickness Dikte
+settings.hitmarker.opacity Ondoorzichtigheid
+settings.hitmarker.leng Lengte
+settings.hitmarker.spacing Spacing
+settings.hitmarker.animD Animatie Grootte
+settings.hitmarker.animS Animatie Snelheid
+settings.hitmarker.hitm Laat zien
+settings.hitmarker.hitmFad Vervaging Snelheid
+settings.hitmarker.custom Hitmarker Afbeelding
+settings.hitmarker.custom.paste Hitmarker Afbeelding URL
+settings.onKillS.style.0 Standaard
+settings.onKillS.style.1 Verminder Resolutie
+settings.onKillS.style.2 Verminder FOV
+settings.onKillS.style.3 Neem FOV toe
+settings.serverb.header Server Browser
+settings.serverb.oldBrowser Legacy Browser
+settings.serverb.defaultRegionOnly Alleen Standaard Regio
+settings.gameprivacy.header Gameplay Privacy
+settings.gameprivacy.streamer Streamer Modus
+settings.gameprivacy.anon Anonieme Modus
+settings.gameprivacy.hideGameTitle Verstop Game Titel
+settings.gameplay.header Gameplay
+settings.gameplay.sensitivityX X Gevoeligheid
+settings.gameplay.sensitivityY Y Govoeligheid
+settings.gameplay.aimsensX Aim X Gevoeligheid
+settings.gameplay.aimsensY Aim Y Gevoeligheid
+settings.gameplay.challenge Challenge Modus
```

Comment:
'Challenge Modus' Challenge is translated into 'Uitdaging/Uitdagings' So 'Challenge Modus' Should be changed to Uitdagings Modus

Reply:
Alright I will change that.